mlflow/pytorch/__init__.py

```diff
@@ -37,6 +37,9 @@
 
 _SERIALIZED_TORCH_MODEL_FILE_NAME = "model.pth"
 _PICKLE_MODULE_INFO_FILE_NAME = "pickle_module_info.txt"
+_TORCHSERVE_ARTIFACTS_KEY = "torchserve_artifacts"
+_TORCHSERVE_EXTRA_FILES_KEY = "extra_files"
+_TORCHSERVE_REQUIREMENTS_FILE_KEY = "requirements_file"
```

Comment:
@shrinath-suresh The torchserve plugin can use these variables to extract required fields from the MLmodel file.